com.woltlab.wcf/templates/recentActivityListItem.tpl

```diff
@@ -22,6 +23,12 @@
 			</div>
 		{/if}
 
+		{if $event->getImage()}
+			<div class="recentActivityListItem__image">
+				<img src="{$event->getImage()}" loading="lazy">
```

Comment:
`loading="lazy"` requires the `height` and `width` attribute to be set, otherwise the browser cannot reserve the required space in advance.